extra/mariabackup/xtrabackup.cc

```diff
@@ -5407,12 +5406,16 @@ static bool xtrabackup_backup_func()
 	}
 	msg("cd to %s", mysql_real_data_home);
 	encryption_plugin_backup_init(mysql_connection);
-	if (innodb_log_checkpoint_now != false && mysql_send_query(
-		    mysql_connection,
+	if (innodb_log_checkpoint_now)
+	{
```

Comment:
This code was formatted to follow the K&R style placement of opening braces. Only in the space-indented code we deviate from that (for anything else than `switch`).